### DISCUSSION TITLE
105?

### DISCUSSION THREAD

**Discussion by AdanGonz2444** [3kh0/ext-remover#611] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Does anyone know any other exploits I can do on version 105? I just downgraded today to do the extension remover, but I don't know if any other exploits will be fun to use on the Chromebook.

**Comment by py660** [comment]
Yes, you can do...
# A LOT.
Once you use extension remover (ingot, etc), you can basically roam free on the internet. Then you *should* use CAUB (see README.md) to prevent your chromebook from updating. After that, you can roam free on the internet, finding cool javascript bookmarklets (on github, for example), and **maybe** find out how to use CVE-2032-2033 to hack the chromebook even more, and **perhaps** use [Chrome Remote Desktop](https://remotedesktop.google.com) to get a full home-computer experience.


DISCLAIMER: DON'T TAKE THIS WITHOUT A GRAIN OF SALT; USE COMMON SENSE BEFORE DOING ANYTHING.

**Reply by JoeMheh** [reply]
What is CVE-2023-2033 exploit I don't think I've heard of it?

**Reply by py660** [reply]
CVE-2023-2033 is a "type confusion" exploit which allows attackers to run malicious code which could potentially bypass the chromeOS's security checks.

**Reply by JoeMheh** [reply]
Oh, so it's still new?

**Reply by py660** [reply]
Yup

**Comment by amelia-bartlett** [comment]
You can also do sh1mmer to at least some extent

**Reply by AdanGonz2444** [reply]
I don't really want to do sh1mmer because I still have to do a few more kiosk exams and I feel like I could get into big trouble with my school which I don't really want to go through. But is it possible to turn on dev mode with sh1mmer? or does sh1mmer only allow to unenroll

**Reply by py660** [reply]
I just have one question: when you downgraded, did your chromebook get completely wiped (management gone), or was it just a version downgrade?

**Reply by AdanGonz2444** [reply, marked as answer]
It was just a version downgrade
